doc/sphinx/source/ops/one_hot.rst

```diff
@@ -15,11 +15,11 @@ Description
 Inputs
 ------
 
-+-----------------+-------------------------+---------------------------------------------------------+
-| Name            | Element Type            | Shape                                                   |
-+=================+=========================+=========================================================+
-| ``arg``         | Any                     | :math:`d_1,\dots,d_{m-1},d_{m+1},\dots,d_n)~(n \geq 0)` |
-+-----------------+-------------------------+---------------------------------------------------------+
++-----------------+-----------------------------+---------------------------------------------------------+
+| Name            | Element Type                | Shape                                                   |
++=================+=============================+=========================================================+
```

Comment:
Non-floating is too general; non-negative integral, or integral if we must.
So adding an `is_integral` would be the right thing.